baseframe/templates/baseframe/components.html

```diff
@@ -11,6 +11,19 @@
   {% endwith %}
 {%- endmacro %}
 
+{% macro flash_messages_material() -%}
+  {% with messages = get_flashed_messages(with_categories=true) %}
+    {% if messages %}
+      {% for category, message in messages %}
```

Comment:
All uses of `{%` should be `{%-` to remove excess whitespace in the rendered template (the original function needs fixing too).

Reply:
Fixed